tests/common.go

```diff
@@ -684,6 +685,45 @@ func ValidatePDSDataServices(ctx *scheduler.Context, errChan ...*chan error) {
 	})
 }
 
+func IsPoolAddDiskSupported() (bool, error) {
+
+	DMthin, err := IsDMthin()
+	if err != nil {
+		return true, nil
+	}
+	if DMthin {
+		pxVersion, err := semver.NewVersion("3.1.0")
```

Comment:
How about we name it dmthinSupportedPxVersion or something similar?

Reply:
good suggestion, changed it

Reply:
My bad. The name "dmthinAddDiskSupportedPxVersion" would have been more appropriate.